xonsh/completer.py

```diff
@@ -467,10 +487,13 @@ def _load_cached_options(self):
         try:
             with open(OPTIONS_PATH, 'rb') as f:
                 self._options = pickle.load(f)
-        except:
+        except Exception:  # pylint: disable=broad-except
             self._options = {}
 
     def _save_cached_options(self):
         """Save completions to file."""
-        with open(OPTIONS_PATH, 'wb') as f:
-            pickle.dump(self._options, f)
+        try:
```

Comment:
> Ahh Ok. that makes a lot more sense.

Which is why I'm starting to think that future lint cleanups should be more focused. I have one more change in the pipeline that is broad; i.e., tries to address most of the lint problems in a given module. But after that I'm going to focus on a subset of problems in each module to keep the review costs as small as possible. That is, I'm no longer going to try and fix a large and disparate set of lint problems in each module in a given commit.